Syl3D/mesh/rectanglemesh.cpp

```diff
@@ -3,14 +3,14 @@
 using namespace mesh;
 
 RectangleMesh::RectangleMesh() {
-    _vertices = new float[] {
+    _vertices = new float[20] {
```

Comment:
Could you change these to 5 * 4 instead of 20, so that people looking at the code will immediately know it has 4 vertices instead of having to count/calculate it?

Reply:
sure

Reply:
Should i do it for other as well ??

Reply:
Yes please